dashboard/app/main.go

```diff
@@ -181,6 +181,10 @@ type uiBug struct {
 	LastActivity   time.Time
 }
 
+func (bug *uiBug) IsFullyPatched() bool {
+	return len(bug.PatchedOn) == bug.NumManagers
```

Comment:
Check bug.Status, if it's patched everywhere we want it will be closed.
We've just discussed closing potentially earlier than it's patched on all managers.
Plus status is more important, if bug.PatchedOn==bug.NumManagers but it's anyhow not closed, we also need to see at at the top.